### PR TITLE
Update stub_app for input, components

### PR DESCRIPTION
**Pull request by gregelin**
Update stub_app used by complianca_app command
or generating compliance app to include "input" and "output" section;
and to have folders for templates, utils, and components.

Add in exception for integrity error to eventually support
gracefully handling.

### CODE REVIEW

File: guidedmodules/stubs/q-files/stub_app/app.yaml [hunk 1]
```diff
@@ -22,3 +22,10 @@ questions:
     title: Example Module
     type: module
     module-id: example
+input:
+  - id: my_input_id
+    name: Input Name
+    type: oscal
+    path: assets/filename.json
```
Comment (davidpofo) [inline guidedmodules/stubs/q-files/stub_app/app.yaml:29]: @gregelin There is no `filename.json` in this PR.

Comment (gregelin) [reply guidedmodules/stubs/q-files/stub_app/app.yaml:29]: The use of non-existent `assets/filename.json` was intentional b/c content will be changed. I guess I could add a sample file with blank information.

Comment (gregelin) [reply guidedmodules/stubs/q-files/stub_app/app.yaml]: Replaced `assets/filename.json` with `components/my_component.json` and added a `my_component.json` file.

**Review by davidpofo — APPROVED** [review]

**Review by davidpofo — APPROVED** [review]
lgtm pr party 4/22

**Pull request merged**